---
layout: default
title: Molecular Beacons
parent: qPCR Methods
has_children: true
nav_order: 3
---

# Molecular Beacons


Molecular beacons are a specialized type of probe used for detecting target sequences in qPCR. They consist of two structural states: an inactive form (loop) and an active form (hybridized to the target sequence). In their inactive loop conformation, molecular beacons do not emit fluorescence. However, when the target sequence is present and sufficient temperature is applied, the beacon melts, anneals to the target sequence, and emits a fluorescence signal.

Unlike hydrolysis probes, molecular beacons are not degraded during amplification. Instead, fluorescence arises from a temperature-induced conformational change (stabilized by the presence of the target sequence). Thus, a molecular beacon cycles from the inactive (loop-forming, unbound) state to the active (denatured and hybridized) state. This difference means that molecular beacons need to be used with polymerases lacking the 5'â†’ 3' exonuclease activity. As the primers are extended beyond the probe sequence, the polymerase displaces the beacon, enabling it to cycle between the aforementioned inactive and active states. This cyclical behavior results in the amplification of the fluorescence signal as the amount of target sequence (=amplicon) increases with each qPCR cycle.

Molecular beacons were first utilized for genotyping by [Kostrikis et al., 1998](<https://www.science.org/doi/abs/10.1126/science.279.5354.1228>). Since then, their adaptability has allowed them to be applied in various (molecular diagnostic) contexts, such as SNP detection (refer to [Mhlanga and Malmberg, 2001](<https://www.sciencedirect.com/science/article/abs/pii/S1046202301912691>)).

![MB.png](Molecular%20Beacons/MB.png)
*Molecular beacon structure and the two alternating conformations - Inactive vs Active*
